src/main/webapp/data.html

```diff
@@ -505,13 +704,76 @@
             <div id="main" class="full-page">
                 <noscript><p class="error">Your browser doesn't appear to support JavaScript. This is most likely because you're using a text-based or otherwise non-graphical browser. Sadly, GSN require javascript in order to work properly. If you want to access directly the data, you can use the api at <a href="http://localhost:22001/gsn">http://localhost:22001/gsn</a>.</p></noscript>
                 <!-- data part -->
-                <br/> <br/> <br/>
+                
                 <div id="datachooser">
                     <div id="data_1" class="clearfix">
                         <form action="multidata" method="post">
 
-                                    <div class="data-output-content">
-                                        <div class="aggregations">
+                                    
+									<div class="left-form-container">
+										<h3>Time range</h3>
+										<div class="time-range">
+                                            <div>
+                                                <span>From</span>
+                                                <input id="datepicker_from" name="from" type="text" value=""   style="width: 150px;float: right;"/>
+                                                <br/>
+                                                <br/>
+                                                <span>To</span>
+                                                <input id="datepicker_to" name="to" type="text" value=""  style="width: 150px; float: right;"/>
+                                            </div>
+                                            <br/>
+                                            <input id="ifrepeat" type="checkbox" name="ifrepeat" value="false" onclick="showRepeatOptions()"/>
+                                            <span>Repeat</span>
+									        
+                                            <input type="number" name="repeat-nmb" id="repeat-number" min="1" max="100" value="1" style="display: none;"/>
+                                            <select name="repeat-type" id="repeat-type" style="display: none;">
+											<option>Years</option>
+											<option>Months</option>
+											<option selected="selected">Days</option>
+											</select>
+                                            <br/>
+                                            <input id="ifexists" type="checkbox" name="ifexists" value="false" checked/>
+                                            <span>Add virtual sensor only if it has data in the given time period</span>
+                                            <br/>
+                                             <div onclick="updateDateList()">Click here for Result: </div><div id="datelist"></div> 
```

Comment:
clickable div should have a "button" appearance or should be visually different (and not use text to indicate their interactions). If it is for debug only, it can be removed for production.